app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -343,56 +335,69 @@ class Contact extends React.Component {
-                      <br />
-                      <span className="font-weight-bold">Afternoon: </span>
-                      <span className="font-weight-light">Between 12:00 and 16:00 in monitoree&apos;s timezone</span>
-                      <br />
-                      <span className="font-weight-bold">Evening: </span>
-                      <span className="font-weight-light">Between 16:00 and 20:00 in monitoree&apos;s timezone</span>
-                    </div>
-                    <Form.Control.Feedback className="d-block" type="invalid">
-                      {this.state.errors['preferred_contact_time']}
-                    </Form.Control.Feedback>
-                  </Form.Group>
-                )}
+                    )}
+                    onChange={e =>
+                      this.handleChange({
+                        target: { id: 'preferred_contact_time', value: e.value },
+                        currentTarget: { id: 'preferred_contact_time' },
+                      })
+                    }
+                    className="mb-1"
+                    styles={preferredContactTimeSelectStyling}
+                    theme={theme => bootstrapSelectTheme(theme, 'lg')}
+                  />
+                  {![null, ''].includes(this.state.current.patient.preferred_contact_time) && (
+                    <React.Fragment>
+                      {[null, ''].includes(this.state.current.patient.preferred_contact_method) &&
+                        this.renderWarningBanner(
+                          'The monitoree will not be sent reminders while they do not have a Preferred Reporting Method selected.',
+                          false,
+                          'warning'
+                        )}
+                      {['Unknown', 'Opt-out'].includes(this.state.current.patient.preferred_contact_method) &&
+                        this.renderWarningBanner(
+                          'The monitoree will not be sent reminders while they have Opt-out or Unknown selected for their Preferred Reporting Method.',
+                          false,
+                          'warning'
+                        )}
+                    </React.Fragment>
+                  )}
+                  <div className="mt-2">
+                    <span className="font-weight-bold">Morning: </span>
+                    <span className="font-weight-light">Between 8:00 and 12:00 in monitoree&apos;s timezone</span>
+                    <br />
+                    <span className="font-weight-bold">Afternoon: </span>
+                    <span className="font-weight-light">Between 12:00 and 16:00 in monitoree&apos;s timezone</span>
+                    <br />
+                    <span className="font-weight-bold">Evening: </span>
+                    <span className="font-weight-light">Between 16:00 and 20:00 in monitoree&apos;s timezone</span>
+                  </div>
+                  <Form.Control.Feedback className="d-block" type="invalid">
```

Comment:
if you put this up under the select, remove the `mb-1` you added and revert to `mt-3` it's the same styling as the primary phone type (which is what we want)